aten/src/ATen/native/DispatchStub.cpp

```diff
@@ -112,7 +112,8 @@ DispatchResult DispatchStubImpl::try_get_call_ptr(
         c10::DeviceType::CUDA,
         c10::DeviceType::HIP,
         c10::DeviceType::MPS,
-        c10::DeviceType::PrivateUse1
+        c10::DeviceType::PrivateUse1,
+        c10::DeviceType::XPU
```

Comment:
The PrivateUse1 should be always at the bottom?

Reply:
sure, has modified.